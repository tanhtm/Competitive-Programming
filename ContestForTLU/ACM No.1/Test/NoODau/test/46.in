825 35
1 1 2 2 4 8 9 10 12 13 15 16 17 18 19 24 24 25 26 26 28 28 28 29 32 34 34 35 35 35 35 37 38 38 39 42 42 43 43 45 50 50 50 51 52 52 53 54 55 55 56 58 60 61 65 67 68 68 69 70 70 74 75 76 76 78 78 79 84 84 85 86 87 87 88 88 90 91 92 94 94 99 100 101 102 106 107 107 107 107 109 110 110 110 111 112 114 115 116 116 120 120 122 125 126 126 127 130 130 137 138 138 139 139 140 141 142 145 145 146 148 150 150 150 152 152 152 154 156 156 157 159 161 161 161 162 163 164 165 165 166 166 168 169 169 170 170 171 172 175 175 177 177 178 179 179 180 180 180 182 182 183 184 186 191 191 192 194 195 197 199 200 200 200 201 202 203 205 207 207 207 209 210 212 212 212 213 214 215 215 219 219 220 220 220 221 223 224 224 225 225 225 226 227 228 228 232 233 235 236 236 237 237 238 241 241 243 243 243 245 247 247 247 248 250 250 251 251 251 254 256 256 257 259 259 260 262 262 265 267 268 271 271 273 277 277 278 280 284 284 285 289 291 291 291 292 294 294 296 299 301 301 304 304 304 305 308 313 314 314 315 316 316 318 318 323 324 327 327 329 331 331 331 331 332 334 338 339 341 347 347 347 350 350 351 354 354 355 356 357 357 358 364 366 368 370 373 375 376 376 376 378 379 380 384 389 391 392 394 396 397 398 399 401 402 402 403 403 407 410 411 411 412 413 415 418 419 420 422 424 424 427 427 429 429 429 430 431 431 431 432 434 435 436 436 439 439 440 440 441 442 443 445 449 450 452 453 453 455 455 455 456 458 459 460 460 461 461 462 464 464 464 465 468 469 470 473 476 477 477 479 480 481 482 482 482 484 485 486 488 492 492 494 495 497 497 499 500 501 502 505 505 506 511 513 515 515 515 516 516 516 517 518 520 520 522 522 527 527 528 528 529 529 534 536 536 541 542 542 543 544 545 545 545 547 548 549 550 550 550 552 554 555 555 555 557 557 558 559 561 562 562 563 564 564 565 566 567 568 571 572 572 572 578 581 583 583 585 585 585 585 586 586 587 588 589 590 590 594 596 596 599 600 601 602 602 603 603 605 606 607 611 611 612 613 614 615 615 615 616 618 619 625 626 628 628 628 628 629 629 630 631 632 632 635 635 636 636 640 645 645 647 648 648 651 651 653 656 657 659 659 660 661 662 664 664 666 667 667 671 672 673 673 675 676 677 678 679 679 680 680 682 683 684 686 687 687 688 689 690 690 691 692 693 694 695 697 698 701 704 706 707 708 709 710 713 715 715 715 717 717 718 721 721 725 731 731 732 733 734 738 740 741 742 743 747 748 749 749 749 751 751 751 751 751 752 753 758 758 761 762 762 765 765 766 767 767 768 768 769 770 770 770 771 771 775 776 776 778 778 779 779 780 784 784 784 785 786 786 787 787 788 789 790 791 792 793 793 796 797 797 798 799 799 799 800 801 802 802 804 805 808 809 809 811 812 815 816 816 817 818 820 821 822 823 823 826 826 827 828 831 831 835 835 836 836 838 838 840 842 843 843 848 850 851 851 852 853 857 858 861 862 864 865 865 866 867 869 870 870 871 873 873 876 880 881 882 884 885 886 887 888 889 890 892 893 894 894 894 899 900 903 903 904 907 909 911 911 915 915 916 918 918 918 920 921 922 925 926 926 928 929 930 933 933 933 934 934 935 935 938 939 940 941 943 944 944 947 947 948 950 951 951 954 954 956 957 957 958 961 961 964 965 966 968 969 971 972 972 973 974 974 975 976 976 980 980 980 980 980 980 980 982 982 983 990 992 992 992 993 994 996 997 999 999 
512 756 40 100 40 442 323 38 868 61 296 798 839 117 333 322 128 143 705 447 900 905 309 698 973 351 259 187 57 758 67 920 514 106 19 
