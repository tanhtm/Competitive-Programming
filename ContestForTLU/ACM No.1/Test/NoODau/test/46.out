415
618
36
83
36
361
276
33
718
54
259
662
699
101
286
276
108
118
581
364
741
745
268
578
800
295
234
165
52
618
56
755
416
86
15
